tests/integration/test_charm.py

```diff
@@ -315,3 +316,17 @@ async def assert_links_in_configmap_by_text_value(
         assert item.text in links_texts
 
     return links_texts
+
+
+@pytest.mark.asyncio
+async def test_dashboard_access(ops_test: OpsTest, lightkube_client: Client):
```

Comment:
can you add a Docstring?

Reply:
Added a docstring with a short explanation